### PR TITLE
Header tooltips

### PR DESCRIPTION
**Pull request by hider**

### CODE REVIEW

File: controller/room.go [hunk 1]
```diff
@@ -18,10 +18,11 @@ func DisplayRoom(c *gin.Context) {
...
 	h := gin.H{
-		"room":    room,
-		"user":    user,
-		"options": model.VoteOptions,
-		"support": consts.Support,
+		"room":     room,
+		"user":     user,
+		"options":  model.VoteOptions,
+		"support":  consts.Support,
+		"hostPort": consts.Domain + consts.Addr,
```
[the remark this answers is not in the document]
Comment (hider) [reply controller/room.go:25]: > For example let's take consts.Domain = example.com while the app could be deployed to pp-next.example.com

Then `pp-next.example.com` should be used as "Domain".

I think both `Domain` and `Addr` should refer to the public address of the application. The container platform can handle the port redirection, thus the internal port can be constant in every launch.

**Comment by hider** [comment]
Here is how I imagine the port handling: [146cc865aaea009939d4df68e37c59397687a73d](https://github.com/garricasaurus/pp-next/pull/26/commits/da5a0bb3db1382ed4299d1e137627b5e14931874)

**Comment by garricasaurus** [comment]
> Here is how I imagine the port handling: [146cc86](https://github.com/garricasaurus/pp-next/commit/146cc865aaea009939d4df68e37c59397687a73d)

I've checked it out and tried it locally, it works with docker.

But how do you imagine the same on Treadmill with LB+DNS alias? Will the share-url label contain the LB'd DNS alias, or the host:port of a single instance?

**Comment by garricasaurus** [comment]
Can you split the PR into 2 separate PRs?

1. header tooltips
2. share url config

I'll merge the first one separately.

**Comment by hider** [comment]
> True, but in my opinion the application should resolve that address at runtime as opposed to having it compiled into the binary. (It should also work with and without docker and display the correct URL).

I agree, I prefer a command line argument or environment variable but you said earlier that you are not a fan that solution.

**Review by garricasaurus — APPROVED** [review]

**Pull request merged**